src/pages/Editor/Classes/EditorUpdateClass.styled.tsx

```diff
@@ -1,18 +1,13 @@
 import styled from 'styled-components';
```

Comment:
i think I updated this file name to be `EditorClasses`, and I think it makes more sense since it's a list of all the classes. `EditorUpdateClass` is like updating the class, which is true, but expecting more like a form to update

Reply:
Updated